common/archive/demultiplexer.go

```diff
@@ -60,9 +60,10 @@ func CreateDemux(namespaceMetadatas []*CollectionMetadata, in io.Reader) *Demult
 		In:              in,
 	}
 	for _, cm := range namespaceMetadatas {
-		ns := cm.Database + "." + cm.Collection
-		demux.NamespaceStatus[ns] = NamespaceUnopened
-
+		if cm.Type != "timeseries" {
```

Comment:
`NamespaceStatus` tracks the status of the demultiplexer reading the data blocks of a specific namespace. There are three options: 

- The demux has not encountered a data block for the namespace (`NamespaceUnopened`)
- The demux has encountered a data block for the namespace (`NamespaceOpened`)
- The demux has encountered the EOF for the namespace (`NamespaceClosed`)

There are no data blocks for the timeseries view, so this means there will be a `NamespaceUnopened` in `demux.NamespaceStatus` at the end of reading the archive. This causes an error in `demux.End()` which expects to see all namespaces in the closed state.

The `NamespaceStatus` for the `system.buckets` collection gets set in `demux.HeaderBSON()` as long as `demux.outs` includes a `DemuxOut` for the `system.buckets` collection. This is always the case since the `DemuxOut` gets added during `demux.Open()`. Therefore it's not necessary to add the `NamespaceStatus` for the `system.buckets` in `CreateDemux()`.

However, this is used as a double check against archive corruption in `demux.End()`. It ensures that we do see the data blocks expected for each metadata prelude. So I think to maintain this check, I'll add a `NamespaceStatus` for the `system.buckets` collection in `CreateDemux()`.